app/components/wallet/hwConnect/common/SaveDialog.scss

```diff
@@ -43,4 +43,10 @@
   }
 }
 
+:global .SaveDialog .Dialog_title {
+  margin-bottom: 12px !important;
+}
 
+:global .SaveDialog .Dialog_actions {
+  margin-top: 24px !important;
+}
```

Comment:
@leivaburto 

The following 
```
:global .SaveDialog .Dialog_title {
  margin-bottom: 12px !important;
}

:global .SaveDialog .Dialog_actions {
  margin-top: 24px !important;
} 
```

is repeated in 3 files,
I think we can set that in a common file like below(in `DialogCommonStyle.scss` without `!important`) :
![image](https://user-images.githubusercontent.com/19986226/61931034-79938680-afba-11e9-8352-88372ecb745c.png)

Reply:
@sarveshwar-singh You are absolutely right ;)